Backpressure Demo has compile error on Windows

Below error happens when compiling backpressure-demo on Windows.

> [INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.951 s
[INFO] Finished at: 2022-01-19T16:10:04+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project backpressure-demo: Could not resolve dependencies for project reactive-grpc-demo:backpressure-demo:jar:1.0-SNAPSHOT: org.openjfx:javafx-graphics:jar:windows:11.0.2 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

I tried modifying this part and confirmed that it compiles normally.
https://github.com/salesforce/reactive-grpc/blob/749eb864054939ba01b7e099a4a2ad435b280dfa/demos/backpressure-demo/pom.xml#L83

`<classifier>win</classifier>`

--- a/demos/backpressure-demo/pom.xml
+++ b/demos/backpressure-demo/pom.xml
@@ -64,7 +64,7 @@
             <dependencies>
                 <dependency>
                     <groupId>org.openjfx</groupId>
-                    <artifactId>javafx-graphics </artifactId>
+                    <artifactId>javafx-graphics</artifactId>
                     <version>11.0.2</version>
                     <classifier>mac</classifier>
                 </dependency>
@@ -78,9 +78,9 @@
             <dependencies>
                 <dependency>
                     <groupId>org.openjfx</groupId>
-                    <artifactId>javafx-graphics </artifactId>
+                    <artifactId>javafx-graphics</artifactId>
                     <version>11.0.2</version>
-                    <classifier>windows</classifier>
+                    <classifier>win</classifier>
                 </dependency>
             </dependencies>
         </profile>

